tools/c7n_azure/c7n_azure/constants.py

```diff
@@ -75,11 +75,13 @@
 ENV_CUSTODIAN_DISABLE_SSL_CERT_VERIFICATION = 'CUSTODIAN_DISABLE_SSL_CERT_VERIFICATION'
 
 """
-Authentication Resource
+Authentication Resource Endpoints
 """
-RESOURCE_ACTIVE_DIRECTORY = 'https://management.core.windows.net/'
-RESOURCE_STORAGE = 'https://storage.azure.com/'
-RESOURCE_VAULT = 'https://vault.azure.net'
+STORAGE_AUTH_ENDPOINT = 'https://storage.azure.com/'
```

Comment:
I'd probably consider different names for these.. It is not an endpoint, maybe some resource type? @stefangordon what do you think?

In the code we assign its value to the `resource` property.. So feels confusing.

Reply:
The endpoints are used to get a token for each resource, so it felt like an auth endpoint. We assign it `auth_endpoint` in the Session constructor, but we assign the `vault` to the `resource` property which I agree is confusing. They are all used to dynamically get an auth endpoint depending on the Cloud. 

`STORAGE_AUTH_ENDPOINT` will stay the same regardless on the cloud. For storage, you pass in the correct suffix. As seen in `storage_utils.py`